src/Model/CustomerAccount.java

```diff
@@ -0,0 +1,123 @@
+package Model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import API.CartService;
+import API.UserService;
+import Model.paymentMethods.Cash;
+import Model.paymentMethods.CreditCard;
+import Model.paymentMethods.FinancialPayment;
+import Model.paymentMethods.PaypalAccount;
+
+
+
+public class CustomerAccount implements CustomerService {
+	
+	protected String userName;
```

Comment:
You should have a Customer entity with these fields:

protected String userName;
+	private CreditCard creditCard;
+	private PaypalAccount paypal;
+	/**
+	 * money that has in his wallet   
+	 */
+	private Cash money;
+	
+	protected Cart myShoppingCart;
+	
+	protected List<CustomerService> friends;
+	protected ProfilePhoto profilePhoto;